README.md

```diff
@@ -15,6 +15,56 @@ List of available parameters:
 
 - `GATSBY_API_URL`: override api url (defaults to location origin)
 
+### MongoDB Setup
+
+Mongo needs a couple of extra steps in order to start a secure cluster.
+
+* Open port 27017 to all nodes that will take part in the cluster.
+* Manually run an initialisation `docker run` with extra environment variables 
+that will initialise the admin user with a password (example below).
+* Manually add a `mgkey` file under `./docker/data/mongo` with the respective 
+secret (see Mongo's keyfile access control for details).
```

Comment:
could link it https://docs.mongodb.com/manual/tutorial/enforce-keyfile-access-control-in-existing-replica-set/